app/src/main/res/values-de/strings.xml

```diff
@@ -109,4 +109,22 @@
   <string name="widget_text_hint_service_data">Entitäts-ID</string>
   <string name="widget_text_hint_service_domain">Domain</string>
   <string name="widget_text_hint_service_service">Dienst</string>
+  <string name="themes_title_settings">Theme</string>
```

Comment:
I think localize is supposed to pick these up for us

Reply:
@dshokouhi  is correct, you need to add any other language that English to Lokalise otherwise they get overridden when we sync the other languages.

Reply:
Ok. Just done it because it was easy to translate right here. I will remove the the translation.